### PR TITLE
Feat/tpac

### PR DESCRIPTION
**Pull request by cavacado**
## Summary

What is the background of this pull request?
add functionality to support v4 for TPAC event.

## Changes

- What are the changes made in this pull request?
- modified certain auxilary functions for verification.

## note:
- this will be reverted once the TPAC event has past.

### CODE REVIEW

File: src/utils/shared.ts [hunk 1]
```diff
@@ -14,23 +14,28 @@ export const getTemplateUrl = (rawDocument: WrappedOrSignedOpenAttestationDocume
   if (utils.isWrappedV2Document(rawDocument)) {
     const documentData = getData(rawDocument);
     return typeof documentData.$template === "object" ? documentData.$template.url : undefined;
-  } else {
+  } else if (utils.isWrappedV3Document(rawDocument)) {
     return rawDocument.openAttestationMetadata.template?.url;
+  } else {
+    return rawDocument.renderMethod?.url;
```
Comment (superical) [inline src/utils/shared.ts:20]: Based on this logic, does this mean v4 will be the default?

**Review by superical — APPROVED** [review]
🙇‍♂️🙇‍♂️🙇‍♂️

**Comment by superical** [comment]
Oh ya... This should also be pushed to `alpha` instead of `master` too, right?

File: src/utils/shared.ts (hunk 1 above)
Comment (cavacado) [reply src/utils/shared.ts:20]: was thinking of just using v4 as the default for this event, then once everything is settled then refactor and put v2 as the fallback in the end. (I never thought so far ahead)

**Pull request merged**